scripts/validate_docstrings.py

```diff
@@ -325,7 +332,9 @@ def validate_all(prefix, ignore_deprecated=False):
             api_items += list(get_api_items(f))
 
     for func_name, _, section, subsection in api_items:
-        if prefix and not func_name.startswith(prefix):
+        if (
+            prefix and not func_name.startswith(prefix)
+        ) or func_name in ignore_functions:
```

Comment:
I'd probably have a separate `if`, this is a bit cumbersome, and looks like black makes it even more unreadable.
```suggestion
        if func_name in ignore_functions:
            continue
        if prefix and not func_name.startswith(prefix):
```